### PR TITLE
Minor documentation edits

### PR DESCRIPTION
**Pull request by dpseidel**
Mostly small edits to links to resolve warnings when rebuilding documentation.
Various other tidying edits the result of #121.

### CODE REVIEW

File: R/colour-mapping.r [hunk 1]
```diff
@@ -373,7 +373,7 @@ filterRange <- function(f) {
 #' @return A function that takes a numeric vector and returns a character vector
 #'   of the same length with RGB or RGBA hex colors.
 #'
-#' @seealso \link[grDevices]{colorRamp}
+#' @seealso \code{\link[grDevices]{colorRamp}}
```
Comment (hadley) [inline R/colour-mapping.r]: I think this should be `[grDevices:colorRamp]`

File: R/pal-dichromat.r [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #' Dichromat (colour-blind) palette (discrete).
 #'
+#' @noMd
```
Comment (hadley) [inline R/pal-dichromat.r]: What do you need this for?

Comment (dpseidel) [reply R/pal-dichromat.r]: without it the \Sexpr that follows breaks - seems to be the same bug documented 
 in klutometis/roxygen#682.

Comment (hadley) [reply R/pal-dichromat.r]: Bummer

**Pull request merged**